cypress-tests/cypress/commands/platform/platformApiCommands.js

```diff
@@ -568,52 +669,72 @@ Cypress.Commands.add(
         userRole = "end-user",
         userPassword = "password",
         workspaceName = "My workspace",
-        metaData = {}
+        metaData = {},
+        groups = []
```

Comment:
- Extract helper functions - Break down the logic into smaller, reusable pieces
- Simplify cookie parsing - Extract token extraction logic
- Remove unused variables - Clean up invitationToken and organizationToken scope issues.

Please conside these.